index.html

```diff
@@ -5,38 +5,16 @@
 <html>
 
         <title><%=settings.title%></title>
-    <script>
```

Comment:
Is removing the license an upstream change?

Reply:
yeah

Reply:
(otherwise would've left as is)